### DISCUSSION TITLE
Oops, that's a four-oh-four. Request failed with status code 500

### DISCUSSION THREAD

**Discussion by erbanku** [spencerwooo/onedrive-vercel-index#825] [Q&A] [answered]
在当前目录包含 `README.md` 文件时无法显示其内容，并出现如图错误。

`Oops, that's a four-oh-four. Request failed with status code 500`

请问这种情况该如何解决呢？

![image](https://user-images.githubusercontent.com/79493862/211710672-12570560-e9ec-44f4-b646-286d4f9da7b5.png)

**Comment by erbanku** [comment]
也看到几个相关讨论，但跟我的情况似乎不一样（相关讨论那个是达到数据库读取限制，我并没有）。

**Comment by spencerwooo** [comment, marked as answer]
https://github.com/spencerwooo/onedrive-vercel-index/discussions/813

本意是将 README 内容用 Vercel 缓存加速加载，但是如果确实出现问题就取消这个设置吧。

**Reply by erbanku** [reply]
按此方法修改后，问题解决了~
